downloads/windows.md

```diff
@@ -37,7 +39,9 @@ current version 2017-10
 [64-bit Download](https://downloads.iridiumbrowser.de/windows/iridiumbrowser-latest-x64.zip "download Portable 64-bit Version"){:.button .small .fit .download1 .icon .fa-download}
 </div>
 **Please note!**        
-Running Iridium portable version works out of the box. As a recommendation we recommend to create a shortcut using the “--user-data-dir=/IridiumUserData” configuration option to store the user data to the preferred location (like a USB stick, etc.)
+<strong>* </strong>= Running Iridium portable version works out of the box. As a recommendation we recommend to create a shortcut using the      
```

Comment:
"as a recommendation we recommend to", uh, why not just write "We recommend to".